lib/workers/global/config/parse/env.ts

```diff
@@ -237,5 +238,28 @@ export async function getConfig(
 
   unsupportedEnv.forEach((val) => delete env[val]);
 
+  config = migratePlatformOptions(config, env);
+  return config;
+}
+
+function migratePlatformOptions(
+  config: AllConfig,
+  env: NodeJS.ProcessEnv,
+): AllConfig {
+  const platformOptionsKeys = ['platformVersion', 'gitLabIgnoreApprovals'];
+  const platformOptions: Record<string, unknown> = {};
+  let updated = false;
+  for (const key of platformOptionsKeys) {
+    const envKey = getEnvName({ name: key });
+    if (!is.undefined(env[envKey])) {
+      updated = true;
+      platformOptions[key] = env[envKey];
+    }
+  }
+
+  if (updated) {
+    config.platformOptions = { ...config.platformOptions, ...platformOptions };
+  }
```

Comment:
see cli changes

Reply:
I modified your suggestions a bit..such that if there are no platform options passed from cli, then an empty object `platformOptions` is not added to the config. Did it to keep the changes to the tests minimum, else almost half the existing changes `cli.spec.ts` would need change.